src/shared/lib/comparison/ComparisonStore.ts

```diff
@@ -1012,6 +1012,59 @@ export default abstract class ComparisonStore extends AnalysisStore
         },
     });
 
+    readonly mutationsEnrichmentDataRequestGroups = remoteData({
+        await: () => [
+            this.alterationsEnrichmentAnalysisGroups,
+            this.selectedStudyMutationEnrichmentProfileMap,
+        ],
+        invoke: () => {
+            if (
+                _(this.selectedMutationEnrichmentEventTypes)
+                    .values()
+                    .some()
```

Comment:
i haven't seen some used like this (without any callback).  it means that there are items in values array?  could we just use length then?

Reply:
this was mostly copied from the alterationsEnrichmentDataRequestGroups and thats what it used. but yes I think it just check if there are items in the values array and we can use length

Reply:
this was mostly copied from alterationsEnrichmentDataRequestGroups and I believe it checks that there is at least one selected event type, or the values array contains a true boolean